backend.native/tests/build.gradle

```diff
@@ -4327,6 +4327,25 @@ dynamicTest("interop_cleaners_leak") {
     flags = ['-Xopt-in=kotlin.native.internal.InternalForKotlinNative']
 }
 
+dynamicTest("interop_migrating_main_thread") {
+    // TODO: Fix it.
```

Comment:
These tests suffer from the same problem as all `dynamicTest`s above. They are getting collectively fixed in https://github.com/JetBrains/kotlin-native/pull/4461